CTP/conf/ha_repl.conf

```diff
@@ -0,0 +1,61 @@
+#Copyright (c) 2016, Search Solution Corporation. All rights reserved.
+##--------------------------------------------------------------------
+##
+##Redistribution and use in source and binary forms, with or without 
+##modification, are permitted provided that the following conditions are met:
+##
+##  * Redistributions of source code must retain the above copyright notice, 
+##    this list of conditions and the following disclaimer.
+##
+##  * Redistributions in binary form must reproduce the above copyright 
+##    notice, this list of conditions and the following disclaimer in 
+##    the documentation and/or other materials provided with the distribution.
+##
+##  * Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products 
+##    derived from this software without specific prior written permission.
+##
+##THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, 
+##INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE 
+##DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, 
+##SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR 
+##SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, 
+##WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
+##USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+# To configure port, password for ssh protocol and test db for default
+default.testdb=xdb
+default.ssh.pwd=<ssh pwd>
+default.ssh.port=<ssh port>
+
+# To configure instance01(master and slave) for host ip, user and broker ports
+env.instance01.master.ssh.host=<master ip>
+env.instance01.master.ssh.user=<ssh user>
+env.instance01.slave1.ssh.host=<slave ip>
+env.instance01.slave1.ssh.user=<ssh user>
+env.instance01.cubrid.cubrid_port_id=<cubrid port>
+env.instance01.ha.ha_port_id=<ha port>
+env.instance01.broker.BROKER_PORT=<broker port>
+
+# If execute continue testing when testing is interrupted
+main.mode.continue=false
+# The execution way on master, it supports jdbc and csql
+main.testmode=jdbc
+# The root path of test case, it should be checked out on controller node
+main.testcase.root=<the path of test cases will be used to test on controller node>
```

Comment:
@xkyu 
actually, main.testbuild.url is not required for user, but if you don't want to set it, you must set main.deploy.rebuild_yn=false, that means your test machine has already installed test build. otherwise, you must set a url which is available for build package for build installation.

Anyway, we will update readme to clarify which must be required. thanks